apps/web/src/components/AccountCard/AccountSelector.tsx

```diff
@@ -26,14 +28,9 @@ export const AccountSelector = () => {
         </Text>
         <Flex alignItems="center" gap="4px">
           <Text color={color("700")} size="sm">
-            {accounts[0].label}
+            {formatPkh(accounts[0].address.pkh)}
```

Comment:
why so? IIRC we should show both the label and the address

Reply:
According to design there should be only account address
<img width="608" alt="image" src="https://github.com/trilitech/umami-v2/assets/169451151/ac9405a6-8749-422f-9758-c10b8ddd718c">